CHANGELOG.md

```diff
@@ -5,20 +5,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [v1.1.0] - 2025-05-01
+
+### Added
+ - New vector interpolations for H(div) and H(curl) spaces (`Nedelec`, `RaviartThomas`,
+   `BrezziDouglasMarini`). ([#1045],[#1162])
+ - New boundary condition type, `ProjectedDirichlet`, for H(div) and H(curl) interpolations. ([#1151])
+   It can be added to a constraint handler the same way as for a regular `Dirichlet`.
+ - Support for exporting discontinuous fields to VTK. ([#867])
+   This happens automatically when a DofHandler with a discontinuous field is used
+   to construct the `VTKGridFile`, but can be requested with the `write_discontinuous`
+   keyword argument.
+
+### Fixes
+ - `addboundaryfacetset` has been fixed for mixed grids. ([#1176])
+ -  `evaluate_at_grid_nodes` now respects the precision of the input dof vector. ([#1044])
 
 ### Removed
- - The deprecated third type parameter for interpolations have been removed. Old code which
+ - The deprecated third type parameter for interpolations has been removed. Old code which
    tries to use three parameters will now throw the somewhat cryptic error:
    ```
    julia> Lagrange{2, RefCube, 1}()
    ERROR: too many parameters for type
    ```
    ([#1083])
```

Comment:
``````suggestion
 - The deprecated third type parameter for interpolations has been removed. ([#1083]) 
   Old code which tries to use three parameters will now throw the somewhat cryptic error:
   ```
   julia> Lagrange{2, RefCube, 1}()
   ERROR: too many parameters for type
   ```
``````

Looks better than current? 
![image](https://github.com/user-attachments/assets/b215dd7f-92c0-446b-89ec-7eaebec9c9b7)